File: Rule.c
#include <stdio.h>

int main() {
    int num;

    printf("Enter a number: ");
    scanf("%d", &num);

    int lastDigit = num % 10;

    if (lastDigit % 2 == 0)
        printf("%d is divisible by 2", num);
    else
        printf("%d is not divisible by 2", num);

    return 0;
}

File: Rule.java
import java.util.Scanner;

public class DivisibleBy2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num = sc.nextInt();

        int lastDigit = num % 10;

        if (lastDigit % 2 == 0)
            System.out.println(num + " is divisible by 2");
        else
            System.out.println(num + " is not divisible by 2");
    }
}
